modules/AlphaWalletFoundation/AlphaWalletFoundation/Types/WalletBalance.swift

```diff
@@ -75,11 +75,7 @@ extension Balance: CustomStringConvertible {
     }
 }
 
-extension WalletBalance: Hashable {
-    public static func == (lhs: WalletBalance, rhs: WalletBalance) -> Bool {
-        return lhs.wallet.address == rhs.wallet.address && lhs.totalAmount == rhs.totalAmount && lhs.change == rhs.change
-    }
-}
+extension WalletBalance: Hashable { }
```

Comment:
It would then compare `etherToken` too though. Is it ever different?

Reply:
yes it will, i don't remember why it was skipped, guess for some prev logic (before adding dataStore),  but for now we need default impl of == operator for struct